oc-includes/osclass/helpers/hUtils.php

```diff
@@ -322,6 +322,7 @@ function osc_get_locations_json() {
  * @return string
  */
 function osc_get_locations_sql($location) {
+    $location = str_replace(' ', '%20', $location);
```

Comment:
Isn't better to use urlencode( $location)

Reply:
I've had problems with URL encode in this situations so I didn't use it.

Reply:
Use rawurlencode it will stick to the standard . Let me know if you have any specific issue with it.

Reply:
@navjottomer That did it. I updated the branch.